src/python/pants/bin/pants_runner.py

```diff
@@ -16,33 +16,25 @@
 logger = logging.getLogger(__name__)
 
 
+@dataclass
 class PantsRunner(ExceptionSink.AccessGlobalExiterMixin):
     """A higher-level runner that delegates runs to either a LocalPantsRunner or
     RemotePantsRunner."""
 
-    def __init__(
-        self,
-        args: Optional[List[str]] = None,
-        env: Optional[Mapping[str, str]] = None,
-        start_time: Optional[float] = None,
-    ) -> None:
-        """
-        :param args: The arguments (sys.argv) for this run. (Optional, default: sys.argv)
-        :param env: The environment for this run. (Optional, default: os.environ)
-        """
-        self._args = args or sys.argv
-        self._env = env or os.environ
-        self._start_time = start_time
+    args: List[str]
+    env: Mapping[
+        str, str,
+    ]
```

Comment:
This is weird formatting that I think is happening because the comma after the second `str`. Can you please remove that comma and try putting this on the same line?